src/index.js

```diff
@@ -52,6 +52,8 @@ const forwardRubocopToLinter =
 
 export default {
   activate() {
+    require('atom-package-deps').install('linter-rubocop', true);
```

Comment:
Just a note, no other provider shows the prompt, but it certainly doesn't hurt to make it explicit to the user what is going on 😉.